docs/design/ctrv_model.md

```diff
@@ -0,0 +1,41 @@
+# Constant turn-rate and velocity (CTRV) model
```

Comment:
Accidently clicked accept in my last PR. This is just a comment so I can click request changes.